tests/ashley/machina_extensions/forum_permission/test_permission_handler.py

```diff
@@ -66,3 +71,58 @@ def test_forum_list(self):
         self.assertNotContains(response, "Forum A2")
         self.assertContains(response, "Forum B1")
         self.assertNotContains(response, "Forum B2")
+
+    def test_get_readable_forums(self):
```

Comment:
I think you also need a test for the case when it is not a queryset but a list?

Reply:
Yep, I added it... and it was worth it :grimacing: